openmrs/apps/registration/registrationCardLayout/printWithBarcode.html

```diff
@@ -40,29 +36,41 @@ <h3 class="print-center">Bahmni Hospital</h3>
                 </tr>
                 <tr>
                     <td class="registrationCard-details-name">
-                        <span class="hindi-text">Age</span>
+                        <span class="hindi-text">{{ "REGISTRATION_AGE" | translate }}</span>
                     </td>
                     <td class="registrationCard-details-seperator"> :  </td>
                     <td>{{patient.age | age}} </td>
                 </tr>
                 <tr>
                     <td class="registrationCard-details-name">
-                        <span class="hindi-text">Gender</span>
+                        <span class="hindi-text">{{ "REGISTRATION_GENDER" | translate }}</span>
                     </td>
                     <td class="registrationCard-details-seperator"> :  </td>
                     <td>{{patient.gender}} </td>
                 </tr>
             
                 <tr>
                     <td class="registrationCard-details-name">
-                        <span class="hindi-text">Village</span>
+                        <span class="hindi-text">{{ "REGISTRATION_ADDRESS" | translate }}</span>
```

Comment:
@prashantsable1991 According to requirements: Address: Only display Area

Area is stored as address1 not address3. Please update.